src/partition.rs

```diff
@@ -931,6 +931,12 @@ impl PredictionMode {
         let max_sample_val = ((1 << bit_depth) - 1) as i32;
         let y_filter_idx = if height <= 4 { 4 } else { 0 };
         let x_filter_idx = if width <= 4 { 4 } else { 0 };
+        let (shift_1, shift_2) = {
```

Comment:
Here you can assign a single variable name for the tuple and access them through `shift.0` and `shift.1`. It might be a bit cleaner.

Reply:
Done